content/docs/guides/contribute/code-reviews/index.fr.md

```diff
@@ -10,4 +10,46 @@ resources:
     byline: "Gunnar Morling / CC-BY-CA 4.0"
 ---
 
+# Vie d'une pull request
+
+L'auteur d'une PR est responsable de sa « vie ». Il se charge de contacter les différents acteurs, de suivre la revue de code, répondre aux commentaires et corriger le code suite à la revue de code (vous pouvez également consulter la [page dédiée à la contribution dans le code](../code/#share-your-changes)).
+
+Le reviewer/mainteneur s'engage à effectuer la revue de code rapidement, c'est aussi à lui qu'appartient de fermer les « request changes », de bien vérifier l'historique des commits, et de fusionner la PR s'il en a les droits.
+
+> En cas de PR conséquente, ne pas hésiter à solliciter plusieurs reviewers qui pourront s'organiser, voire de faire la review ensemble, reviewers et auteur.
+
+Voici une proposition de flux de travail, les guillemets indiquent la **nécessité** de communiquer afin de garantir le fonctionnement du flux de la validation d'une PR :
+
+```mermaid
+sequenceDiagram
+  actor A as Auteur PR
+  actor R as Reviewer/mainteneur
+  
+  A->>R: DEMANDE « je voudrais une review »
+  R->>A: RÉPOND ✅ « Ok » ou ❌ « Pas ok »
+  
+  R-->>+A: COMMENTE, demande des changements
+  A-->>R: RÉPOND aux commentaires
+  A-->>R: RÉPOND aux requests changes
+  A-->>-R: FAIT les modifications nécessaires dans des « fixups » dédiés
+  R-->>A: REVOIE à nouveau le code et FERME les request changes et conversations
+ 
+  A->>R: FAIT rebase & fixup
+  R->>A: VÉRIFIE les commits
+  R->>A: DIT ✅ « C'est approuvé. »
+  Note left of R: FUSIONNE si mainteneur
+```
+
+> Si la personne qui fait la revue de code n'est pas mainteneur, l'auteur de PR a la charge de trouver un mainteneur pour pouvoir fusionner sa PR avec le code. Dans certains cas particuliers (notamment juste avant le gel des fusions en fin d'itération), le mainteneur doit être averti plus tôt.
+
+# Revue de code
+
+Voici quelques conseils et recommandations qui nous semblent pertinentes pour une revue de code humaine, pertinente qui édifie tous ses contributeurs.
```

Comment:
nit: more natural to me
```suggestion
Voici quelques conseils et recommandations qui nous semblent pertinentes pour une revue de code humaine, pertinente et enrichissante tous ses contributeurs.
```
(not sure about the 🇬🇧 version either)

Reply:
You're right! Done.

Reply:
Done